docs/pelion_ready/device-management-config.md

```diff
@@ -84,13 +86,19 @@ To run the tests:
 $ mbed test -t <TOOLCHAIN> -m <BOARD> -n simple*dev*connect --run -v
 ```
 
-## 2. Bootloader configuration
+## Bootloader configuration
 
-After you've successfully passed the "Connect" tests as described above, you can enable firmware update feature by adding a bootloader to your application.
+When the "connect" tests have passed, you can enable the firmware update feature by adding a bootloader to your application.
 
-1. Import as a new application the official [mbed-bootloader](https://github.com/ARMmbed/mbed-bootloader/) repository or the [mbed-bootloader-extended](https://github.com/ARMmbed/mbed-bootloader-extended/) repository that builds on top of `mbed-bootloader` and extends the support for file systems and storage drivers. You can do this with `mbed import mbed-bootloader-extended`.
+1. Import the official [mbed-bootloader](https://github.com/ARMmbed/mbed-bootloader/) repository as a new application:
 
-1. Inside the imported bootloader application, and edit the application configuration, for example `mbed-bootloader-extended/mbed_app.json`. Add a new target entry similar to the step above, and specify:
+    `mbed import mbed-bootloader`
+  
+    Alternatively, you can import the [mbed-bootloader-extended](https://github.com/ARMmbed/mbed-bootloader-extended/) repository that builds on top of `mbed-bootloader` and extends the support for file systems and storage drivers:
+  
+    `mbed import mbed-bootloader-extended`.
+
+1. In the imported bootloader application, edit the configuration, for example `mbed-bootloader-extended/mbed_app.json`. Add a new target entry and specify:
```

Comment:
"entry, and"

Reply:
Academic comma dilemma. I'd rather not have that.

Reply:
I don't think it's academic; +1 for adding it.